set.mm

```diff
@@ -389523,6 +389523,48 @@ then any point of the half line ( ` R ` ` A ` ) also lies opposite to
        $)
   $}
 
+  ${
+    dfcgrg2.p $e |- P = ( Base ` G ) $.
+    dfcgrg2.m $e |- .- = ( dist ` G ) $.
+    dfcgrg2.g $e |- ( ph -> G e. TarskiG ) $.
+    dfcgrg2.a $e |- ( ph -> A e. P ) $.
+    dfcgrg2.b $e |- ( ph -> B e. P ) $.
+    dfcgrg2.c $e |- ( ph -> C e. P ) $.
+    dfcgrg2.d $e |- ( ph -> D e. P ) $.
+    dfcgrg2.e $e |- ( ph -> E e. P ) $.
+    dfcgrg2.f $e |- ( ph -> F e. P ) $.
+    dfcgrg2.1 $e |- ( ph -> A =/= B ) $.
+    dfcgrg2.2 $e |- ( ph -> B =/= C ) $.
+    dfcgrg2.3 $e |- ( ph -> C =/= A ) $.
+    $( Congruence for two triangles can also be defined as congruence of sides
+       and angles (6 parts).  This is often the actual textbook definition of
+       triangle congruence, see for example
+~ https://en.wikipedia.org/wiki/Congruence_(geometry)#Congruence_of_triangles
+       With this definition, the "SSS" congruence theorem has an additional
+       part, namely, that triangle congruence implies congruence of the length
+       of sides.  Because our development of elementary geometry strives to
```

Comment:
In my previous comment, there was the problem with "each", that you fixed, but the main point was that you can write "congruence of the sides" or "equality of the lengths of sides", but not "congruence of the lengths of sides" (since congruence of segments means equality of their lengths).

Maybe you can write:
> namely, that triangle congruence implies congruence of the sides (which means equality of their lengths).

or something like that.

Anyway, you get the point and you'll find the better phrasing (since I haven't read the whole previous discussion in the issue).